tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -655,11 +691,11 @@ void test_constraint_energy_analytic(const Solution& solution,
   const auto four_index_constraint =
       GeneralizedHarmonic::four_index_constraint(d_phi);
   const auto two_index_constraint = GeneralizedHarmonic::two_index_constraint(
-      d_gauge_function, normal_one_form, normal_vector, inverse_spatial_metric,
+      d4_gauge_function, normal_one_form, normal_vector, inverse_spatial_metric,
       inverse_spacetime_metric, pi, phi, d_pi, d_phi, gamma2,
       three_index_constraint);
   const auto f_constraint = GeneralizedHarmonic::f_constraint(
-      gauge_function, d_gauge_function, normal_one_form, normal_vector,
+      gauge_function, d4_gauge_function, normal_one_form, normal_vector,
```

Comment:
Should `d_gauge_function` be renamed `d3_gauge_function` to make it obvious it is 3d?

Reply:
fixed this consistently, see response [above](https://github.com/sxs-collaboration/spectre/pull/2528#discussion_r504874946)